templates/event_agent_delete.conf.j2

```diff
@@ -0,0 +1,14 @@
+# {{ ansible_managed }}
+[event-agent]
+tube = {{ openio_event_agent_tube_delete }}
+namespace = {{ openio_event_agent_namespace }}
+user = openio
+queue_url = {{ openio_event_agent_queue_url }}
+workers = 1
```

Comment:
must be a variable!
this is the only valid reason to set this configuration apart